test/unit/builtins/object.spec.ts

```diff
@@ -121,3 +121,93 @@ describe(".hasOwnProperty()", () => {
         `.expectToMatchJsResult();
     });
 });
+
+const trueFalseTests = [[true], [false]] as const;
```

Comment:
Used to be true, false and undefined

But these two cases aren't possible to replicate in Lua since you can't check is a property exists and is set to undefined

```ts
const foo = { bar: true };
Object.defineProperty(foo, "bar", { configurable: undefined }); // configurable -> false
Object.defineProperty(foo, "bar", {}); // configurable -> true
```